### PR TITLE
Misc. typo & whitespace fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../fc-word-whitelist.txt --skip="*.ts,*.po,./src/3rdParty,./src/Mod/Assembly/App/opendcm,./.git,./src/zipios++"`

### CODE REVIEW

File: src/Mod/Arch/Presets/pset_definitions.csv [hunk 1]
```diff
@@ -26,7 +26,7 @@ Pset_AudioVisualApplianceTypeCommon;Reference;IfcIdentifier
 Pset_AudioVisualApplianceTypeDisplay;NominalSize;IfcPositiveLengthMeasure;DisplayWidth;IfcPositiveLengthMeasure;DisplayHeight;IfcPositiveLengthMeasure;Brightness;IfcIlluminanceMeasure;ContrastRatio;IfcPositiveRatioMeasure;RefreshRate;IfcFrequencyMeasure;VideoResolutionWidth;IfcInteger;VideoResolutionHeight;IfcInteger
 Pset_AudioVisualApplianceTypePlayer;PlayerMediaEject;IfcBoolean
 Pset_AudioVisualApplianceTypeProjector;VideoResolutionWidth;IfcInteger;VideoResolutionHeight;IfcInteger
-Pset_AudioVisualApplianceTypeSpeaker;Impedence;IfcFrequencyMeasure
+Pset_AudioVisualApplianceTypeSpeaker;Impedance;IfcFrequencyMeasure
```
Comment (yorikvanhavre) [inline src/Mod/Arch/Presets/pset_definitions.csv]: This shoudn't be changed, it's not a comment, it comes from the IFC format. If we changed it we'll loose compatibility

Comment (luzpaz) [reply src/Mod/Arch/Presets/pset_definitions.csv]: reverted commit. Will try to submit something upstream at some point.

**Comment by luzpaz** [comment]
Rebased with master.

**Comment by yorikvanhavre** [comment]
merged

**Pull request closed without merging**